src/messaging/ExchangeMgr.h

```diff
@@ -107,7 +107,7 @@ class DLL_EXPORT ExchangeManager : public SecureSessionMgrDelegate
      *                                                             is full and a new one cannot be allocated.
      *  @retval #CHIP_NO_ERROR On success.
      */
-    CHIP_ERROR RegisterUnsolicitedMessageHandler(uint32_t protocolId, ExchangeDelegate * delegate);
+    CHIP_ERROR RegisterUnsolicitedMessagesHandler(uint32_t protocolId, ExchangeDelegate * delegate);
```

Comment:
Is the plural `Messages` intentional change?

Reply:
Yes, there are two reasons:

1. If we overload the following two functions:
    CHIP_ERROR RegisterUnsolicitedMessageHandler(uint32_t protocolId, ExchangeDelegate * delegate);
    CHIP_ERROR RegisterUnsolicitedMessageHandler(MessageType msgType, ExchangeDelegate * delegate);

err = mpExchangeMgr->RegisterUnsolicitedMessagesHandler(Protocols::kProtocol_InteractionModel, this); will confuse, and call into second one from my test, which is not what we want.

2. CHIP_ERROR RegisterUnsolicitedMessagesHandler(uint32_t protocolId, ExchangeDelegate * delegate); is to register a handler for a group of messages, this should be reflected in the function name.